lib/bots/syncbot.ts

```diff
@@ -39,10 +39,11 @@ export class SyncBot extends ProcBot {
 	 * @param to         Definition {service, flow} of the flow being emitted to.
 	 * @param messenger  Service to use to interact with the cloud.
 	 * @param logger     Logger to use to interact with the maintainers.
+	 * @param actions    An array of which actions to sync along this route.
 	 * @returns          Function that processes events from the messenger.
 	 */
 	private static makeRouter(
-		from: FlowDefinition, to: FlowDefinition, messenger: MessengerService, logger: Logger
+		from: FlowDefinition, to: FlowDefinition, messenger: MessengerService, logger: Logger, actions: MessengerAction[],
```

Comment:
NIT: Splitting this up so that there is one parameter per line would make it a lot more readable